install_config/aggregate_logging.adoc

```diff
@@ -1682,7 +1682,8 @@ Where `$espod` is the name of any one of your Elasticsearch pods.
 Curator allows administrators to configure scheduled Elasticsearch maintenance
 operations to be performed automatically on a per-project basis. It is scheduled
 to perform actions daily based on its configuration. Only one Curator pod is
-recommended per Elasticsearch cluster. Curator is configured via a YAML
+recommended per Elasticsearch cluster. Curator pods only run at the time stated
+in the `cronjob` and then the pod terminates. Curator is configured via a YAML
```

Comment:
s/`cronjob`/CronJob ??
A quick search in Atom seems to show CronJob is the preferred spelling, with no ``. There are a few cronjobs.

Reply:
On that same page, I see it as all lower case, "cronjob" with no backticks. I'll remove the backticks and make it match the rest of the page. Thanks!